osf_tests/factories.py

```diff
@@ -59,8 +59,11 @@ class UserFactory(DjangoModelFactory):
     fullname = factory.Sequence(lambda n: 'Freddie Mercury{0}'.format(n))
 
     username = factory.LazyFunction(fake_email)
-    password = factory.PostGenerationMethodCall('set_password',
-                                                'queenfan86')
+    password = factory.PostGenerationMethodCall(
+        'set_password',
+        'queenfan86',
+        notify=False
+    )
```

Comment:
Does this change fix any test failures? If so, what is/are it/they?

Reply:
There's probably a few more examples, but here's some of the fails:
```
 osf_tests/test_institution.py:139: in test_send_deactivation_email_call_count
    assert mock_send_mail.call_count == 2
E   AssertionError: assert 4 == 2
E    +  where 4 = <MagicMock name='send_mail' id='139986975588704'>.call_count
------------------------------ Captured log call -------------------------------
INFO     osf.models.institution:institution.py:222 Institution deactivation notification email has been sent to [2/2] users for [6320aa24405bf3176806b47c]
_ TestOSFGroup.test_notify_group_member_email_does_not_send_before_throttle_expires _
osf_tests/test_osfgroup.py:224: in test_notify_group_member_email_does_not_send_before_throttle_expires
    assert mock_send_mail.call_count == 1
E   AssertionError: assert 2 == 1
E    +  where 2 = <MagicMock name='send_mail' id='139986965797744'>.call_count
___ TestOSFGroup.test_notify_group_member_email_sends_after_throttle_expires ___
osf_tests/test_osfgroup.py:240: in test_notify_group_member_email_sends_after_throttle_expires
    assert mock_send_mail.call_count == 1
E   AssertionError: assert 2 == 1
E    +  where 2 = <MagicMock name='send_mail' id='139986963925536'>.call_count
_________ TestOSFGroup.test_notify_group_unregistered_member_throttle __________
osf_tests/test_osfgroup.py:258: in test_notify_group_unregistered_member_throttle
    assert mock_send_mail.call_count == 1
E   AssertionError: assert 2 == 1
E    +  where 2 = <MagicMock name='send_mail' id='139986964755176'>.call_count
________________ TestUser.test_visible_contributor_to_property _________________
osf_tests/test_user.py:1826: in test_visible_contributor_to_property
    assert project_to_be_invisible_on._id not in visible_contributor_to_nodes
E   AssertionError: assert 'jhkpf' not in ['dm69n', 'jhkpf']
E    +  where 'jhkpf' = (title='Mandatory user-facing encoding', category='project') with guid 'jhkpf'._id
________________ TestUserMerging.test_merge_doesnt_send_signal _________________
osf_tests/test_user.py:2103: in test_merge_doesnt_send_signal
    assert mock_notify.called is False
E   AssertionError: assert True is False
E    +  where True = <MagicMock name='send_mail' id='139986980667744'>.called

``